docs/public-networks/reference/cli/options.md

```diff
@@ -2916,25 +2916,25 @@ service.
 === "Syntax"
 
     ```bash
-    --rpc-http-authentication-jwt-public-key-file=<FILE>
+    --rpc-ws-authentication-jwt-public-key-file=<FILE>
     ```
 
 === "Example"
 
     ```bash
-    --rpc-http-authentication-jwt-public-key-file=publicKey.pem
+    --rpc-ws-authentication-jwt-public-key-file=publicKey.pem
     ```
 
 === "Environment variable"
 
     ```bash
-    BESU_RPC_HTTP_AUTHENTICATION-JWT-PUBLIC-KEY-FILE="publicKey.pem"
+    BESU_RPC_WS_AUTHENTICATION_JWT_PUBLIC_KEY_FILE="publicKey.pem"
```

Comment:
Probably out of scope of this PR, but I don't see any references to this env var in besu and so not sure how it will actually work...maybe there's some magic dynamic variable name thing somewhere :/

Reply:
Aye, its supposed to be dynamic from memory and has the BESU prefix in front of everything. This would have been put in circa late 2019 I think. Can change it back or alias if its an issue?

Reply:
yeah it's resolved dynamically - see EnvironmentVariableDefaultProvider